src/test/java/org/sqlite/DBMetaDataTest.java

```diff
@@ -1669,4 +1667,593 @@ public void getPrimaryKeys(String table) throws SQLException {
+            stat.executeUpdate(
+                    "create table db2.REFERRING (ID integer, RID integer, constraint fk\r\n foreign\tkey\r\n(RID) "
+                            + "references REFERRED(id))");
+
+            exportedKeys = meta.getExportedKeys(null, "db2", "referred");
+            assertThat(exportedKeys.getString("PKTABLE_NAME")).isEqualTo("REFERRED");
+            assertThat(exportedKeys.getString("FKTABLE_NAME")).isEqualTo("REFERRING");
+            assertThat(exportedKeys.getString("FK_NAME")).isEqualTo("fk");
+            exportedKeys.close();
+        }
+
+        @Test
+        public void getIndexInfoOnTestForAttachedDatabase() throws SQLException {
+            ResultSet rs = meta.getIndexInfo(null, "db2", "test2", false, false);
+            assertThat(rs).isNotNull();
+        }
+
+        @Test
+        public void getIndexInfoIndexedSingleForAttachedDatabase() throws SQLException {
+            stat.executeUpdate(
+                    "create table db2.testindex (id integer primary key, fn float default 0.0, sn not null);");
+            stat.executeUpdate("create index db2.testindex_idx on testindex (sn);");
+
+            ResultSet rs = meta.getIndexInfo(null, "db2", "testindex", false, false);
+            ResultSetMetaData rsmd = rs.getMetaData();
+
+            assertThat(rs).isNotNull();
+            assertThat(rsmd).isNotNull();
+        }
+
+        @Test
+        public void getIndexInfoIndexedSingleExprForAttachedDatabase() throws SQLException {
+            stat.executeUpdate(
+                    "create table db2.testindex (id integer primary key, fn float default 0.0, sn not null);");
+            stat.executeUpdate("create index db2.testindex_idx on testindex (sn, fn/2);");
+
+            ResultSet rs = meta.getIndexInfo(null, "db2", "testindex", false, false);
+            ResultSetMetaData rsmd = rs.getMetaData();
+
+            assertThat(rs).isNotNull();
+            assertThat(rsmd).isNotNull();
+        }
+
+        @Test
+        public void getIndexInfoIndexedMultiForAttachedDatabase() throws SQLException {
+            stat.executeUpdate(
+                    "create table db2.testindex (id integer primary key, fn float default 0.0, sn not null);");
+            stat.executeUpdate("create index db2.testindex_idx on testindex (sn);");
+            stat.executeUpdate("create index db2.testindex_pk_idx on testindex (id);");
+
+            ResultSet rs = meta.getIndexInfo(null, "db2", "testindex", false, false);
+            ResultSetMetaData rsmd = rs.getMetaData();
+
+            assertThat(rs).isNotNull();
+            assertThat(rsmd).isNotNull();
+        }
+
+        @AfterEach
+        public void exit() throws SQLException {
+            stat.executeUpdate("Detach database db2;");
+            testDB.deleteOnExit();
+        }
+    }
+
+    private void assertSystemSchema(ResultSet rs) throws SQLException {
```

Comment:
shouldn't that also check for `TABLE_SCHEM` and compare it to a parameter passed in the function?